apps/demos/Demos/Form/SmartPaste/jQuery/index.js

```diff
@@ -0,0 +1,250 @@
+    xs: 2,
+    sm: 2,
+    md: 2,
+    lg: 2,
+  };
+  const stylingMode = 'filled';
+
+  const form = $('#form').dxForm({
+    formData: {},
+    aiIntegration,
+    labelMode: 'outside',
+    showColonAfterLabel: false,
+    labelLocation: 'top',
+    minColWidth: 220,
+    items: [{
+      itemType: 'group',
+      caption: 'Billing Summary',
+      colCountByScreen,
+      items: [{
+        dataField: 'Amount Due',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          placeholder: '$0.00',
+          stylingMode,
+        },
+        aiOptions: {
+          instruction: 'Format as the following: $0.00',
+        },
+      }, {
+        dataField: 'Statement Date',
+        editorType: 'dxDateBox',
+        editorOptions: {
+          placeholder: 'MM/DD/YYYY',
+          stylingMode: 'filled',
+        },
+        aiOptions: {
+          instruction: 'Format as the following: MM/DD/YYYY',
+        },
+      }],
+    }, {
+      itemType: 'group',
+      caption: 'Billing Information',
+      colCountByScreen,
+      items: [{
+        dataField: 'First Name',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          stylingMode,
+        },
+      }, {
+        dataField: 'Last Name',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          stylingMode,
+        },
+      }, {
+        dataField: 'Phone Number',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          placeholder: '(000) 000-0000',
+          stylingMode,
+        },
+        aiOptions: {
+          instruction: 'Format as the following: (000) 000-0000',
+        },
+      }, {
+        dataField: 'Email',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          stylingMode,
+        },
+        validationRules: [{ type: 'email' }],
+        aiOptions: {
+          instruction: 'Do not fill this field if the text contains an invalid email address. A valid email is in the following format: email@example.com',
+        },
+      }]
+    }, {
+      itemType: 'group',
+      caption: 'Billing Address',
+      colCountByScreen,
+      items: [{
+        dataField: 'Street Address',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          stylingMode,
+        },
+      }, {
+        dataField: 'City',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          stylingMode,
+        },
+      }, {
+        dataField: 'State/Province/Region',
+        editorType: 'dxTextBox',
+        editorOptions: {
+          stylingMode,
+        },
+      }, {
+        dataField: 'ZIP',
+        editorType: 'dxNumberBox',
+        editorOptions: {
+          stylingMode,
+          mode: 'text',
+          value: null,
+        },
+        aiOptions: {
+          instruction: 'If the text does not contain a ZIP, determine the ZIP code from the provided address.',
+        },
+      }]
+    }, {
+      itemType: 'group',
+      cssClass: 'buttons-group',
+      colCountByScreen,
```

Comment:
```suggestion
      colCount: 2,
```

Reply:
with just colCount it will be one column on small screens, but by design we should have two columns everywhere